### ISSUE TITLE
Great repo, though why long name?

### ISSUE THREAD

**Issue by pankajpatel** [harryhope/hooked-on-redux#4]
Awesome work and I think I have been waiting for something like this without realizing

Though why `useHookedOn...` and `hooked-on-redux`?

why not names like `redux-hooks` and `useReduxState`?

Just wanna understand the naming bit better. Thanks 🙂

**Comment by harryhope** [comment]
All of the good wordplays of redux were already taken so I opted for a ridiculous sounding one instead :p 

As for why I didn’t call this `redux-hooks` or something, while it does use redux under the hood, the actual use is nothing like normal redux. With redux I expect to see reducers, actions, action creators, constants, etc... 

Hooked on Redux requires none of them. A codebase that implements hooked on redux will probably look very different from one that implements normal redux (there will be much less code and boilerplate in the former). Because of that I felt it was important to disambiguate the state hook calls especially.

In short with hooked on redux you get _redux the npm  package_ but not _redux the philosophy_, and I tried to name it accordingly: Something that signals that you are using Redux but different enough that it’s clear it’s something different as well.

**Comment by pankajpatel** [comment]
Thanks for the explanation. Maybe you can pin this issue just in case anybody else has this question 😸
